### PR TITLE
Nojira | @ericbakenhus | Adjust old story handling

### PR DESCRIPTION
**Pull request by ericbakenhus**
# READY FOR REVIEW

# Summary
- Adjusting some old event handling and fixing an id

# Review By (Date)
- Whenever

# Criticality
- High

# Review Tasks

## Setup tasks and/or behavior to test

1. Look at code

### CODE REVIEW

File: netlify/functions/event-import-background.mts [hunk 1]
```diff
@@ -228,8 +228,42 @@ export default async (req: Request) => {
...
+                  parent_id: eventArchiveFolderId,
+                },
+              });
+              await delay();
+            }
+            console.log('Done!');
+            console.log('Processing complete: ', id);
+            continue;
+          }
+
```
Comment (ericbakenhus) [inline netlify/functions/event-import-background.mts:266]: Old events are being cleared out of the Google sheets. We'll ask about it, but we should handle that case.

File: netlify/functions/event-import-background.mts [hunk 2]
```diff
@@ -289,27 +325,26 @@ export default async (req: Request) => {
...
-          }
-
-          if (isOld) {
-            console.log('Story is old. Moving...');
-            if (run) {
               await storyblokManagement.put(`/spaces/${spaceId}/stories/${storyblok.id}`, {
                 story: { 
                   ...storyblok,
-                  full_slug: `events/sync-archive/${storyblok.slug}`,
+                  parent_id: eventArchiveFolderId,
```
Comment (ericbakenhus) [inline netlify/functions/event-import-background.mts:336]: Fixing this. Management API needs parent id.

File: netlify/functions/event-import-background.mts (hunk 1 above)
Comment (sherakama) [reply netlify/functions/event-import-background.mts:266]: Huh, I definitely remember that being an expectation/assumption. Thanks for following up on it.

**Review by sherakama — APPROVED** [review]

**Comment by ericbakenhus** [comment]
Really need some refactoring here. This is some wet ass code.

**Comment by sherakama** [comment]
😆 I think that is the first time I have seen the words 'code' 'ass' and 'wet' used in that combination.

**Pull request merged**